nextcord/ext/commands/core.py

```diff
@@ -386,6 +397,31 @@ def __init__(self, func: Union[
         else:
             self.after_invoke(after_invoke)
 
+        # Attempt to bind to parent hooks if applicable
+        if not kwargs.get("inherit_hooks", False) or not self.parent:
```

Comment:
Wouldnt it make more sense for it to error if it doesn't have a parent?

Reply:
No, unless I misunderstand how the internals work.

``self.parent`` is None in the case of things like ``@bot.command()`` (fairly sure) so it ignores those rather than throwing an error was my line of thinking

Reply:
@TAG-Epic